testsuite/smoke-test/tests/forge-aptos.rs

```diff
@@ -19,10 +15,6 @@ fn main() -> Result<()> {
             &ErrorReport,
             &GasCheck,
             &MintTransfer,
-            &ModulePublish,
```

Comment:
If so, we need to use the PackagePublish test because ModulePublish becomes obsolete. This however can currently not be tested because package publishing is turned off when running against real nodes (that is why it was commented out). We should get this (or something equivalent) back once those things settled.